### PR TITLE
Cleanup config server doc

### PR DESCRIPTION
**Pull request by hmusum**
Move uncommon config doc to bottom
zkCli -> vespa-zkcli
Expand and simplify doc on how to scale up and down cluster size
Remove stale zookeeper port doc

### CODE REVIEW

File: en/cloudconfig/configuration-server.html [hunk 1]
```diff
@@ -93,33 +52,31 @@ <h2 id="redundancy">Redundancy</h2>
...
+<a href="../setting-vespa-variables.html">VESPA_CONFIGSERVERS</a> must be set on all nodes.
 This is a comma or whitespace-separated list with the hostname of all configservers,
 like <em>myhost1.mydomain.com,myhost2.mydomain.com,myhost3.mydomain.com</em>.
 </p><p>
-When there are multiple config servers, the config clients
-(usually the <a href="../reference/config-proxy.html">config proxy</a>, if you have not configured the client to use another config source)
-will pick a config server based on a hash of their hostname and how many config servers there are.
-This uniformly loads the config servers.
-The config clients are fault-tolerant and will switch to another config server if it is unavailable
+When there are multiple config servers, the config <a href="../reference/config-proxy.html">config proxy</a>
```
Comment (kkraune) [inline en/cloudconfig/configuration-server.html:64]:
```suggestion
When there are multiple config servers, the <a href="../reference/config-proxy.html">config proxy</a>
```

File: en/cloudconfig/configuration-server.html [hunk 2]
```diff
@@ -140,18 +97,19 @@ <h3 id="scaling-up">Scaling Up</h3>
...
 <h4 id="scaling-up-by-majority">Scaling up by Majority</h4>
 <p>
 When increasing from 1 to 3 nodes or 3 to 7, the blank nodes constitutes a majority in the cluster.
-After restarting the config servers, they will not always contain old application data,
+After restarting the config servers, they will not contain the old application data,
 because the blank nodes might win the election - depending on restart timing.
-Get a new correct data set when repating <em>vespa-deploy prepare</em> and <em>vespa-deploy activate</em>.
-If you do not wish to scratch your old application data like this,
-for instance to keep the history, the solution is to scale up by minor sets of the nodes - example:
+To avoid any issues with this scale up by minor sets of the nodes - example:
```
Comment (kkraune) [inline en/cloudconfig/configuration-server.html:112]:
```suggestion
To avoid any issues with this, scale up by minor sets of the nodes - example:
```

**Review by kkraune — APPROVED** [review]
💯

**Comment by hmusum** [comment]
Thanks! Merging with suggestions

**Pull request merged**